docs/pages/layout/index.vue

```diff
@@ -0,0 +1,154 @@
+      <p>Each Learning Equality product should utilize a grid system, and generally apply the guidelines from <DocsExternalLink href="https://material.io/design/layout/responsive-layout-grid.html#" text="Material Design" />.</p>
+      <h3>
+        Nested grids
+      </h3>
+      <p>
+        A page can contain nested grids. A container component can exist on a grid and also have its own grid that its contained components can lie on.
+      </p>
+      <img src="./nested.png">
+      <h3>
+        Fixed column width
+      </h3>
+      <p>
+        By default, margins have a fixed width, and column width changes responsively. When the need arises, columns can be a fixed width, and margins can change responsively.
+      </p>
+      <video width="600" controls loop>
+        <source src="./fixed-column.mp4" type="video/mp4">
+        Your browser does not support the video tag
+      </video>
+      <h3>
+        Breakpoints
+      </h3>
+      <p>
+        Page layouts should be designed for each of the breakpoint ranges below.
+      </p>
+      <table>
+        <tr>
+          <th>Range</th>
+          <th><DocsExternalLink href="https://v15.vuetifyjs.com/en/framework/grid" text="Vuetify" /></th>
+          <th>KGrid</th>
+          <th>Columns</th>
+          <th>Margins & Gutters</th>
+          <th>Padding</th>
+        </tr>
+        <tr>
+          <td>0-479px</td>
+          <td>xs</td>
+          <td>Level 0</td>
+          <td>4</td>
+          <td>16px</td>
+          <td>8px 16px 16px</td>
+        </tr>
+        <tr>
+          <td>480-599px</td>
+          <td>xs</td>
+          <td>Level 1</td>
+          <td>4</td>
+          <td>16px</td>
+          <td>8px 16px 16px</td>
+        </tr>
+        <tr>
+          <td>600-839px</td>
+          <td>sm</td>
+          <td>Level 2</td>
+          <td>8</td>
+          <td>16px</td>
+          <td>8px 24px 24px</td>
+        </tr>
+        <tr>
+          <td>840-959px</td>
+          <td>sm</td>
+          <td>Level 3</td>
+          <td>12</td>
+          <td>16px</td>
+          <td>8px 24px 24px</td>
+        </tr>
+        <tr>
+          <td>960-1263px</td>
+          <td>md</td>
+          <td>Level 4</td>
+          <td>12</td>
+          <td>24px</td>
+          <td>8px 24px 24px</td>
+        </tr>
+        <tr>
+          <td>1264-1430px</td>
+          <td>lg</td>
+          <td>Level 5</td>
+          <td>12</td>
+          <td>24px</td>
+          <td>8px 24px 24px</td>
+        </tr>
+      </table>
+      <p>Define <DocsExternalLink href="https://material.io/design/layout/spacing-methods.html#spacing" text="keylines" /> in increments of 4px or 8 px when UI elements must be placed outside of the alignment to columns and rows within a grid.</p>
+    </DocsPageSection>
+
+
+  </DocsPageTemplate>
+
+</template>
+
+
+<style lang="scss" scoped>
+
+  td,
+  th {
+    padding: 8px;
+    text-align: left;
+    vertical-align: top;
+  }
+
+</style>
```

Comment:
Not sure if it's important but Github made a point to mention that there is no EOL newline at the end of the file. I thought that our linter typically complained about this.

Reply:
I believe in Kolibri that's handled by some pre-commit hooks, not `kolibri-tools`